Makefile

```diff
@@ -77,6 +77,15 @@ configgen:
 	echo "" >> conf/st2.conf.sample
 	. $(VIRTUALENV_DIR)/bin/activate; python ./tools/config_gen.py >> conf/st2.conf.sample;
 
+.PHONY: .generate-openapi-spec
```

Comment:
I just used what's used on Circle CI, but I can update both places to use the new binary to make it consistent.